docs/Admin.Kafka.md

```diff
@@ -0,0 +1,85 @@
+# Admin API Function: *_kafka* #
+
+----
+
+**Go to:** &nbsp;&nbsp;&nbsp;&nbsp; [**Root TOC**](CM-Well.RootTOC.md) &nbsp;&nbsp;&nbsp;&nbsp; [**Previous Topic**](Admin.Backpressure.md)
+
+----
+
+## Description ##
+
+The **_kafka** API allows you to consume a copy of a CM-Well Kafka queue, which contains messages produced by ingest requests. You may want to use this API for some task that involves monitoring, replicating or testing CM-Well's operations on ingestion requests. (For example, the **_kafka** API is used internally to support the Data Consistency Crawler feature, which allows CM-Well to test each ingested infoton for data consistency.)
+
+CM-Well maintains and consumes 4 Kafka **topics** (named queues):
+
+- **persist_topic** - each message represents an infoton to be written to Cassandra storage.
+- **persist_topic.priority** - same as above, for a request sent with high priority.
+- **index_topic** - each message represents an infoton to be indexed in Elastic Search.
+- **index_topic.priority** - same as above, for a request sent with high priority.
+
+Each request to ingest an infoton produces two messages, one that enters one of the **persist** queues, and one that enters one of the **index** queues.
+
+Each queue can be distributed among several **partitions**. A partition is a portion of the queue maintained on a specific CM-Well node.
+
+When you use the **_kafka** API, in the URL you refer to the specific topic name and partition number you want to consume.
+
+## Kafka Message Formats ##
+
+Each message returned via the **_kafka** API is in a separate line.
+
+When you consume the messages from the 4 Kafka topics described, CM-Well automatically deserializes them into JSON-formatted strings.
+
+If you are consuming a different topic (YAAKOV, LIKE WHAT???), and you know that its messages are also strings, you can specify the **format=text** parameter value to cause its messages to be deserialized as strings. Otherwise, each message is returned as comma-separated byte values (YAAKOV, why do we need the commas?).
```

Comment:
We need the commas to represent the byte values as a list. e.g.
`72,101,108,108,111,32,87,111,114,108,100`